src/sql/upgrade-83.sql

```diff
@@ -0,0 +1 @@
+drop index if exists IndexBuildOutputsOnPath;
```

Comment:
Note that this was introduced in a migration and doesn't seem to have been properly recorded in hydra.sql.

Reply:
I added a comment to that effect in this migration.